package.json

```diff
@@ -4,14 +4,10 @@
   "description": "Codebase for https://vanillaframework.io",
   "main": "_site/index.html",
   "scripts": {
-    "help": "gulp help",
-    "gulp": "gulp",
-    "browser-sync": "gulp browser-sync",
     "build": "gulp build",
```

Comment:
Yep. In this project, Jekyll does the serving, not node. So node doesn't worry itself about `serve`, because it's handled outside.